package.json

```diff
@@ -21,12 +21,16 @@
   "bin": {
     "create-stencil": "index.js"
   },
+  "dependencies": {
```

Comment:
Installing these led to the conversion of `package-lock.json` from the v1 format to the v2 format.  I have yet to find a 'good' way to convert from v1->v2, so I figured we'd just get it over with now. That's the reason for so much churn/change in that file in this PR